content/admin.py

```diff
@@ -1,12 +1,36 @@
 from django.contrib import admin
 from .models import OurStory, OurTeam, EduProgram
-# Register your models here.
+from django.contrib.flatpages.admin import FlatpageForm, FlatPageAdmin
+from django.contrib.flatpages.models import FlatPage
+from tinymce.widgets import TinyMCE
+
+class MCEFlatPageForm(FlatpageForm):
+
+    class Meta:
+        model = FlatPage
+        widgets = {
+            'content' : TinyMCE(attrs={'cols': 100, 'rows': 15}),
+        }
+        exclude = []
+
+class MCEFlatPageAdmin(FlatPageAdmin):
+    """
+    Page Admin
+    """
+    form = MCEFlatPageForm
+
+
+admin.site.unregister(FlatPage)
+admin.site.register(FlatPage, MCEFlatPageAdmin)
```

Comment:
all this and the settings doesnt have anything to do with helper text, right?

Can you redo this PR with only the helper text stuff?